src/config.rs

```diff
@@ -156,6 +161,11 @@ impl OperatorConfig {
         self.server_port = port;
         self
     }
+
+    pub fn resync_period(mut self, period: Option<Duration>) -> Self {
```

Comment:
just a nit, but I think perhaps this name could be more misinterpreted as meaning some sort of regular period for automated resync. Maybe `error_backoff_max`? Or something more along those lines.